### PR TITLE
Moving Feature Labels

### LINKED ISSUE #72: Feature Labels
Fix feature labels from overlapping (e.g. shift up the molecules to separate from lines). Also normalisation slider height on website (mobile is fine).
Separate features from L to T dwarfs.

### PR DESCRIPTION
**Pull request by Will-Cooper**
Closes #72

### CODE REVIEW

**Comment by kelle** [comment]
- I like how you offset the lines from the bands.
- i think there are still too many bands shown. They overlap too much. I vote for getting rid of water and H2. Lemme see if I can comment on the code directly.

File: simple_app/plots.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     'feh': {'altname': [], 'label': r'FeH', 'type': 'band',
             'wavelengths': [[0.8692, 0.875], [0.98, 1.03], [1.19, 1.25],
                             [1.57, 1.64]]},
-    'h2': {'altname': ['cia h2'], 'label': r'H$_2$', 'type': 'band', 'wavelengths': [[1.5, 2.4]]},
+    'h2': {'altname': ['cia h2'], 'label': r'H₂', 'type': 'band', 'wavelengths': [[1.5, 2.4]]},
```
Comment (kelle) [inline simple_app/plots.py:30]:
```suggestion
```
this is too big of a feature so I vote to not display it.

File: simple_app/plots.py [hunk 2]
```diff
@@ -161,7 +161,10 @@ def normalise() -> np.ndarray:
         return None, None, nfail, failstr
     bounds = [normminwave, normmaxwave, fluxmin, fluxmax]
     p.add_tools(HoverTool(tooltips=[('Wave', '@wave'), ('Flux', '@flux')], renderers=lineplots))
-    features = ['k', 'na', 'feh', 'tio', 'co']
+    featuresall = {'L Dwarf Features': ['k', 'na', 'feh', 'tio', 'co', 'h2o', 'h2'],
+                   'T Dwarf Features': ['k', 'ch4', 'h2o', 'h2'],
```
Comment (kelle) [inline simple_app/plots.py:165]:
```suggestion
    featuresall = {'L Dwarf Features': ['k', 'na', 'feh', 'tio', 'co', 'h2o'],
                   'T Dwarf Features': ['k', 'ch4', 'h2o'],
```

**Review by kelle — CHANGES_REQUESTED** [review]
Remove CIA H2 label.

**Review by dr-rodriguez — APPROVED** [review]

**Review by kelle — APPROVED** [review]

**Pull request merged**